pkg/api/persistentvolumeclaim/util_test.go

```diff
@@ -210,90 +210,3 @@ func TestDropDisabledSnapshotDataSource(t *testing.T) {
 		}
 	}
 }
-
-// TestPVCDataSourceSpecFilter checks to ensure the DropDisabledFields function behaves correctly for PVCDataSource featuregate
-func TestPVCDataSourceSpecFilter(t *testing.T) {
-	apiGroup := ""
-	validSpec := core.PersistentVolumeClaimSpec{
-		DataSource: &core.TypedLocalObjectReference{
-			APIGroup: &apiGroup,
-			Kind:     "PersistentVolumeClaim",
-			Name:     "test_clone",
-		},
-	}
-	validSpecNilAPIGroup := core.PersistentVolumeClaimSpec{
-		DataSource: &core.TypedLocalObjectReference{
-			Kind: "PersistentVolumeClaim",
-			Name: "test_clone",
-		},
-	}
-
-	invalidAPIGroup := "invalid.pvc.api.group"
-	invalidSpec := core.PersistentVolumeClaimSpec{
-		DataSource: &core.TypedLocalObjectReference{
-			APIGroup: &invalidAPIGroup,
-			Kind:     "PersistentVolumeClaim",
-			Name:     "test_clone_invalid",
-		},
-	}
-
-	var tests = map[string]struct {
-		spec        core.PersistentVolumeClaimSpec
-		gateEnabled bool
-		want        *core.TypedLocalObjectReference
-	}{
-		"enabled with empty ds": {
-			spec:        core.PersistentVolumeClaimSpec{},
-			gateEnabled: true,
-			want:        nil,
-		},
-		"enabled with invalid spec": {
-			spec:        invalidSpec,
-			gateEnabled: true,
-			want:        nil,
-		},
-		"enabled with valid spec": {
-			spec:        validSpec,
-			gateEnabled: true,
-			want:        validSpec.DataSource,
-		},
-		"disabled with invalid spec": {
-			spec:        invalidSpec,
-			gateEnabled: false,
-			want:        nil,
-		},
-		"disabled with valid spec": {
-			spec:        validSpec,
-			gateEnabled: false,
-			want:        nil,
-		},
-		"diabled with empty ds": {
-			spec:        core.PersistentVolumeClaimSpec{},
-			gateEnabled: false,
-			want:        nil,
-		},
-		"enabled with valid spec but nil APIGroup": {
```

Comment:
IMO all "enabled" tests are still valid and should be preserved. The best would be to remove `gateEnabled bool` and leave the tests that have it as `true`.

Reply:
Oops!  That was a mistake on my side; I'll get those back in place.  Thanks for catching this.